tests/test_helpers/test_assistants.py

```diff
@@ -92,24 +92,24 @@ def test_AIAssistant_invoke():
     messages = thread.messages.order_by("created_at").values_list("message", flat=True)
     messages_ids = thread.messages.order_by("created_at").values_list("id", flat=True)
 
-    assert response_0 == {
-        "history": [],
-        "input": "What is the temperature today in Recife?",
-        "output": "The current temperature in Recife today is 32 degrees Celsius.",
-    }
-    assert response_1 == {
-        "history": [
-            HumanMessage(content="What is the temperature today in Recife?", id=messages_ids[0]),
-            AIMessage(
-                content="The current temperature in Recife today is 32 degrees Celsius.",
-                id=messages_ids[1],
-            ),
-        ],
-        "input": "What about tomorrow?",
-        "output": "The forecasted temperature in Recife for tomorrow, June 10, 2024, is "
-        "expected to be 35 degrees Celsius.",
-    }
-    assert list(messages) == messages_to_dict(
+    assert response_0["input"] == "What is the temperature today in Recife?"
+    assert response_0["output"] == "The current temperature in Recife today is 32 degrees Celsius."
+    assert response_1["input"] == "What about tomorrow?"
+    assert (
+        response_1["output"]
+        == "The forecasted temperature for tomorrow in Recife is 35 degrees Celsius."
+    )
+
+    question_message = response_1["messages"][1]
+    assert question_message.content == "What is the temperature today in Recife?"
+    assert question_message.id == str(messages_ids[0])
+    response_message = response_1["messages"][2]
+    assert (
+        response_message.content == "The current temperature in Recife today is 32 degrees Celsius."
+    )
+    assert response_message.id == str(messages_ids[1])
```

Comment:
I preferred the old approach for readability. If you had trouble with the diffs, it's possible to change pytest to include all the diff.

Reply:
Yes, it was better, the problem is that in the new version of langchain-core there's a bunch of metadata (see example bellow) that is not relevant to tests added to messages:
<img width="651" alt="Screenshot 2024-09-11 at 14 11 45" src="https://github.com/user-attachments/assets/9ded4c0d-06ad-493f-8156-f683262277d8">

Reply:
Got it! We could use something like this: https://kalnytskyi.com/posts/assert-str-matches-regex-in-pytest/
In this case, it would be a comparator that always returns true.
But that's a nitpick.